x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClusterSpecIT.java

```diff
@@ -208,6 +207,32 @@ static CsvSpecReader.CsvTestCase convertToRemoteIndices(CsvSpecReader.CsvTestCas
             var newFrom = "FROM " + remoteIndices + " " + commands[0].substring(fromStatement.length());
             testCase.query = newFrom + query.substring(first.length());
         }
+        if (commands[0].toLowerCase(Locale.ROOT).startsWith("metrics")) {
+            String command = commands[0];
+            int startIndex = "metrics".length();
+            while (startIndex < command.length() && Character.isWhitespace(command.charAt(startIndex))) {
+                startIndex++;
```

Comment:
Nit: would this be simpler with String.split?

Reply:
Yes, I pushed https://github.com/elastic/elasticsearch/pull/109593/commits/55bc904f91c469408c28f63b88c8859935e233c7 to simplify this.